pom.xml

```diff
@@ -91,6 +91,30 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
Sorry for not adding a note -- I added this configuration to disable the plugin. However, the plugin is already included in this build. It looks [like this is in the parent plugin](https://github.com/jenkinsci/plugin-pom/blob/plugin-3.4/pom.xml#L557).

The failures are unrelated to my change, too.

If you run `mvn help:effective-pom` you can see the definition. I can't for the life of me find out why this is being pulled in:

If you are not seeing this behavior, could it be because of some environment configuration disabling the same?

Reply:
It looks like I can disable this with a property instead. I'll push that fix.

Reply:
It looks like upstream has an updated plugin parent pom, and this parent now uses 'spotbugs'. It found bugs unrelated to my change so I've added the spotsbugs.skip property to allow `mvn clean install` to pass